src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -334,8 +334,8 @@ static Solution UpdateExistingDocumentsToChangedDocumentContents(Solution soluti
         /// name="transformation"/> as it will have its <see cref="Solution.WorkspaceVersion"/> updated
         /// accordingly.</param>
         private protected (Solution oldSolution, Solution newSolution) SetCurrentSolution<TData>(
-            Func<Solution, TData, Solution> transformation,
             TData data,
+            Func<Solution, TData, Solution> transformation,
```

Comment:
made the 'data' parameter the first parameter.  makes working with the api much nicer as you see what data you're passing in prior to the static lambda that uses it.